packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.ts

```diff
@@ -25,11 +25,11 @@ export const appBadges = `
   <mj-section>
     <mj-column>
       <% if (locals.onDesktopOrTabletDevice) { %>
-      <mj-text css-class="secondary-text mb-3"><span data-l10n-id="another-desktop-device"> Or, install on
+      <mj-text css-class="secondary-text mb-3 mt-2"><span data-l10n-id="another-desktop-device"> Or, install on
```

Comment:
This seems like such a small thing, but since the images themselves already have `mb-3`, can we just increase that to `mb-5` and remove `mt-2` here? If there's elements one after another, I try to keep required margins between them on one or the other element instead of a margin-bottom on one and a margin-top on the other. Since we reuse these app badges in other templates, it'll ensure that text after that is always `mb-5` across our templates, rather than having to add `mt-2` to text elements that follow (I think if you left this as-is, it'd require you adding `mt-2` to text elements in `postVerify` to match the spacing).

Reply:
Ohhh yeah I didn't think about other templates, but thanks for such a detailed explanation :)